modules/net-lb-app-ext/README.md

```diff
@@ -45,13 +45,13 @@ module "glb-0" {
   backend_service_configs = {
     default = {
       backends = [
-        { backend = "projects/myprj/zones/europe-west8-b/instanceGroups/myig-b" },
-        { backend = "projects/myprj/zones/europe-west8-c/instanceGroups/myig-c" },
+        { backend = module.ig-zone-a.group },
+        { backend = module.ig-zone-b.group },
       ]
     }
   }
 }
-# tftest modules=1 resources=5
+# tftest modules=3 resources=9 fixtures=ig
```

Comment:
Let's add e2e here, so we can check that it works for both cases

Reply:
done. how can we test it?